src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -1,19 +1,23 @@
 (ns puppetlabs.trapperkeeper.internal
-  (:import (clojure.lang ExceptionInfo IFn IDeref))
+  (:import (clojure.lang ExceptionInfo IFn IDeref)
+           (java.lang ArithmeticException NumberFormatException)
+           (java.io FileNotFoundException))
   (:require [clojure.tools.logging :as log]
             [beckon]
             [plumbing.graph :as graph]
             [slingshot.slingshot :refer [throw+]]
             [puppetlabs.kitchensink.core :refer [add-shutdown-hook! boolean? cli!]]
-            [puppetlabs.trapperkeeper.config :refer [config-service]]
+            [puppetlabs.trapperkeeper.config :refer [config-service get-in-config]]
             [puppetlabs.trapperkeeper.app :as a]
             [puppetlabs.trapperkeeper.common :as common]
             [puppetlabs.trapperkeeper.services :as s]
             [puppetlabs.kitchensink.core :as ks]
             [schema.core :as schema]
             [clojure.core.async :as async]
             [clojure.core.async.impl.protocols :as async-prot]
-            [clojure.string :as string]))
+            [clojure.string :as string]
+            [me.raynes.fs :as fs]
+            [clojure.string :as str]))
```

Comment:
`clojure.string` is already required as `string`, so requiring it again doesn't seem necessary.